src/main/java/Parser.java

```diff
@@ -0,0 +1,139 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Parser {
+    
+    Parser() {
+        
+    }
+
+    /**
+     * Helps to parse commands from the scanner, that tells the bot what actions
+     * to take
+     * @param list
```

Comment:
The description of parameters should be included. Similar problems are found in other places.